pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -137,6 +158,9 @@ func (pfactory *PredicateMetadataFactory) GetMetadata(pod *v1.Pod, nodeNameToInf
 	if pod == nil {
 		return nil
 	}
+	// existingPodSpreadConstraintsMap represents how existing pods matches "pod"
```

Comment:
pods matches -> pods match

Reply:
I will update in the original PR which first introduced this change, and rebase here.